ios/deploy_ios.sh

```diff
@@ -1,27 +1,19 @@
-#!/bin/sh -e
+#!/bin/bash
+set -eu
 
-# autobuild_2_build: actual build process
-
-
-####################
-###  PARAMETERS  ###
-####################
-
-
-source "$(dirname "${BASH_SOURCE[0]}")/../ensure_THIS_JAMULUS_PROJECT_PATH.sh"
-
-###################
-###  PROCEDURE  ###
-###################
-
-cd "${THIS_JAMULUS_PROJECT_PATH}"
+## Builds an ipa file for iOS. Should be run from the repo-root
 
+# Create Xcode file and build
 qmake -spec macx-xcode Jamulus.pro
 /usr/bin/xcodebuild -project Jamulus.xcodeproj -scheme Jamulus -configuration Release clean archive -archivePath "build/Jamulus.xcarchive" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
-# Make a deploy folder
-mkdir deploy
+
 # Generate ipa by copying the .app file from the xcarchive file
-mkdir deploy/Payload
-cp -r build/Jamulus.xcarchive/Products/Applications/Jamulus.app deploy/Payload/
-cd deploy
+mkdir build/Payload
+cp -r build/Jamulus.xcarchive/Products/Applications/Jamulus.app build/Payload/
+cd build || exit 1
```

Comment:
`set -eu` should mean the `|| exit 1` isn't needed -- it's effectively the same thing, as far as I'm aware.

Reply:
Hmm, [shellcheck made me do this](https://github.com/koalaman/shellcheck/wiki/SC2164) and I assumed that `cd` (as a shell built-in) was special compared to other commands which I know are checked with `set -e`.
I've run a quick test and it looks you are right. I'm going to remove the `|| exit 1`.

This is what happened (probably): I took the script, ran shell check, noted the warning, fixed it and only later noticed that the script did not use `set -eu` and added that. So, rather PEBKAC than a shellcheck bug... :)